packages/core/src/views/account-center/Maximized.svelte

```diff
@@ -44,6 +44,8 @@
   )
 
   const { position } = state.get().accountCenter
+
+  $: innerWidth = 0
```

Comment:
```suggestion
  let innerWidth = 0
```
This can be a regular let binding rather than a reactive one.